addons/web/static/tests/views/kanban_view_tests.js

```diff
@@ -12925,4 +12925,38 @@ QUnit.module("Views", (hooks) => {
         );
         assert.containsN(target, ".o_kanban_group .o_kanban_record", 16);
     });
+    QUnit.test("Keep scrollTop when loading records with load more", async (assert) => {
+        for (let id = 1; id <= 50; id++) {
+            serverData.models.partner.records.push({
+                id,
+                bar: false,
+                display_name: `record ${id}`,
+            });
+        }
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch: `<kanban>
+                <templates>
+                <t t-name="kanban-box">
+                    <div style="height:250px;"><field name="id"/></div>
```

Comment:
height here not necessary/not working, right?

Reply:
Currently, it's working (on my local with Qunit.debug) and I think setting a bigger high can make me avoid having to add some mocked records (I try this).